packages/inter-protocol/test/swingsetTests/psmUpgrade/bootstrap-psm-upgrade.js

```diff
@@ -0,0 +1,313 @@
+// @ts-check
+
+import { Fail, NonNullish } from '@agoric/assert';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import { CONTRACT_ELECTORATE, ParamTypes } from '@agoric/governance';
+import { deeplyFulfilledObject, makeTracer } from '@agoric/internal';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import { mustMatch } from '@agoric/store';
+import { makeBoard } from '@agoric/vats/src/lib-board.js';
+import { makeRatio } from '@agoric/zoe/src/contractSupport/ratio.js';
+import { PaymentPKeywordRecordShape } from '@agoric/zoe/src/typeGuards.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+import { makePromiseKit } from '@endo/promise-kit';
+import { withAmountUtils } from '../../supports.js';
+
+const trace = makeTracer('BootPSMUpg');
+
+export const faV1BundleName = 'psmV1';
```

Comment:
s/fa/psm/g. 

you had me confused for a while. I was trying to figure out why it was working with the fluxAggregator.  Particularly since the minor edit to a fluxAggregator test was in the same commit, so I found it first when searching for `faV1BundleName`!

Reply:
sorry bout that. I thought I rename those but maybe lost in git